docs/source/usersguide/install.rst

```diff
@@ -42,37 +42,32 @@ the environment, run:
 
     conda activate openmc-env
 
-.. _install_ppa:
+-------------------------------------------
+Installing on Linux/Mac/Windows with Docker
+-------------------------------------------
 
------------------------------
-Installing on Ubuntu with PPA
------------------------------
-
-For users with Ubuntu 15.04 or later, a binary package for OpenMC is available
-through a `Personal Package Archive`_ (PPA) and can be installed through the
-`APT package manager`_. First, add the following PPA to the repository sources:
-
-.. code-block:: sh
-
-    sudo apt-add-repository ppa:paulromano/staging
-
-Next, resynchronize the package index files:
+OpenMC can be easily deployed using `Docker <https://www.docker.com/>`_ on any
+Windows, Mac, or Linux system. With Docker running, execute the following
+command in the shell to download and run a `Docker image`_ with the most recent
+release of OpenMC from `DockerHub <https://hub.docker.com/>`_ called
+``openmc/openmc:v0.10.0``:
 
 .. code-block:: sh
 
-    sudo apt update
+    docker run openmc/openmc:v0.10.0
```

Comment:
Same as above. Although maybe there is no 0.12.1 container for Docker yet?

Reply:
Thanks for pointing this out -- new releases should update the `latest` tag on Docker, so I've put that instead